src/index.ts

```diff
@@ -0,0 +1,53 @@
+export { default as Deck } from './components/deck';
+export { default as Slide, SlideContext } from './components/slide/slide';
+export { Appear, Stepper } from './components/appear';
+export { default as CodePane } from './components/code-pane';
+export {
+  OrderedList,
+  Quote,
+  Heading,
+  ListItem,
+  UnorderedList,
+  Text,
+  Link,
+  CodeSpan
+} from './components/typography';
+export {
+  Table,
+  TableCell,
+  TableRow,
+  TableHeader,
+  TableBody
+} from './components/table';
+export type { TableProps } from './components/table';
```

Comment:
Rabbit hole here ...
to avoid compiler warnings, I had to use `export type` here...
to avoid prettier warnings, I had to upgrade to `prettier@2.x`
which requires arrow functions to use parenthesis (eg `(p) => p` not `p => p`)
which made tiny changes to about 100 files

So ...
sorry for such noise!

Reply:
@carlos-kelly Great, another problem: eslint@8 dropped support for Node@10.
Our PR checks run against Node 10/12/14/16.  
Should we drop Node 10 support?  (It's, like, SUUUPER old)

Reply:
@scottrippey -- Yes, please update to 12, 14, 16

Reply:
Done!